remotespark/RemoteSparkMagics.py

```diff
@@ -62,11 +62,16 @@ def sparkmagic(self, line, cell=""):
             client_to_use = self.client_manager.get_client(args.client)
 
         # Execute
-        print(self._send_command(client_to_use, command, args.sql))
+        result = self._send_command(client_to_use, command, args.sql)
 
         # Revert mode
         Log.mode = previous_mode
 
+        if result is str:
```

Comment:
I don't think we need this conditional. A returned string should be displayed by default.